### DISCUSSION TITLE
metals Ram sie

### DISCUSSION THREAD

**Discussion by tsxjsx** [scalameta/metals-vscode#1468] [Q&A]
Is there any server property available to reduce the Metal process?

**Comment by tgodzik** [comment]
`metals.serverProperties` 1g is set there by default. I wouldn't go lower than 500m

**Comment by tsxjsx** [comment]
Even after setting the maximum heap size (-Xmx) to 500m, the Metals process continues to consume nearly 1GB of memory.
and the the metal process execute the child process scala-cli get above 1gb when i start debug but the metals server proeprtie xmx1G not applied to this child process 
how can i customize the proeprties for scal cli process and simlarly for bloop process executed by metals classpath process

**Comment by tgodzik** [comment]
By the way this is a diplicate discussion https://github.com/scalameta/metals-vscode/discussions/1460

There is already some info in the other one.
